.pubnub.yml

```diff
@@ -15,6 +15,7 @@ changelog:
     changes:
       - type: bug
         text: add functions to (fetchMessages and deleteMessages) to declaration files for TS
+    date: Aug 6, 2018
```

Comment:
Please note that there is a date format check now. This is not valid anymore, the format it accepts will be 2018-08-06

Reply:
Right. It's now switched to the format you mentioned